lib/libcheri_caprevoke/Makefile

```diff
@@ -0,0 +1,15 @@
+SRCS= libcheri_caprevoke.c
+
+NEED_COMPAT=64C
+
+INCS= libcaprevoke.h
+INCSDIR=	${INCLUDEDIR}/cheri
+
+CFLAGS+= -I${.CURDIR}
+CFLAGS+= -I${.CURDIR}/${TARGET_ARCH}
```

Comment:
You shouldn't be using TARGET_* here, but MACHINE_*.  Also, if you use 'MACHINE_CPUARCH' that will just be 'riscv' on all riscv architectures avoiding the need for the symlink.  (That's also a more typical practice in the tree when things are shared among sub-architectures).  There is a table in arch(7) that shows the value of the various MACHINE* make variables for each architecture.

Reply:
Yes, thanks.